app/uploaders/attachment_uploader.rb

```diff
@@ -4,7 +4,7 @@ class AttachmentUploader < WhitehallUploader
 
   THUMBNAIL_GENERATION_TIMEOUT = 10.seconds
   FALLBACK_PDF_THUMBNAIL = File.expand_path("../assets/images/pub-cover.png", __dir__)
-  EXTENSION_WHITELIST = %w[chm csv diff doc docx dot dxf eps gif gml ics jpg kml odp ods odt pdf png ppt pptx ps rdf ris rtf sch txt vcf wsdl xls xlsm xlsx xlt xml xsd xslt zip].freeze
+  EXTENSION_BLOCKLIST = %w[chm csv diff doc docx dot dxf eps gif gml ics jpg kml mp4 odp ods odt pdf png ppt pptx ps rdf ris rtf sch txt vcf wsdl xls xlsm xlsx xlt xml xsd xslt zip].freeze
```

Comment:
You're changing two things in one commit, it would be better to have two separate commits, or at least describe the second change as well.

I don't think blocklist is a suitable substitute for whitelist, as the semantics are reversed.

Reply:
Good points